client/src/pages/Homepage/Homepage.tsx

```diff
@@ -0,0 +1,46 @@
+import useStyles from './useStyles';
+import { Box, Grid, TextField, InputLabel, Button } from '@material-ui/core';
+import homepage_image from '../../Images/homepage_image_v2.png';
+export default function Homepage(): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Box className={classes.wrapper}>
+      <Grid container className={classes.gridContainer}>
+        <Grid item xs={12} sm={6} className={classes.contentWrapper}>
+          <form className={classes.form}>
+            <h1>Find the care your dog deserves</h1>
```

Comment:
Please use MUI `Typogarphy` component for this `h1`

Reply:
resolved by commit 8cf73207227fbc2ac8aa4489e71b28120227c004